commands/plugin_e2e_test.go

```diff
@@ -0,0 +1,145 @@
+package commands
+
+import (
+	"bytes"
+
+	"github.com/mattermost/mattermost-server/v5/model"
+	"github.com/spf13/cobra"
+
+	"github.com/mattermost/mmctl/client"
+	"github.com/mattermost/mmctl/printer"
+)
+
+const (
+	pluginID            = "com.mattermost.demo-plugin"
+	pluginURL           = "https://github.com/mattermost/mattermost-plugin-demo/releases/download/v0.8.0/com.mattermost.demo-plugin-0.8.0.tar.gz"
+	nonExistentPluginID = "nonExistentPluginID"
+)
+
+func (s *MmctlE2ETestSuite) TestPluginEnableCmd() {
+	s.SetupTestHelper().InitBasic()
+	installPlugin(s, pluginID, pluginURL)
+	defer removePlugin(s, pluginID)
+
+	s.RunForSystemAdminAndLocal("Successful enable plugin", func(c client.Client) {
+		printer.Clean()
+
+		appErr := s.th.App.DisablePlugin(pluginID)
+		s.Require().Nil(appErr)
+
+		plugins, appErr := s.th.App.GetPlugins()
+		s.Require().Nil(appErr)
+		s.Require().Len(plugins.Active, 0)
+		s.Require().Len(plugins.Inactive, 1)
+
+		cmd := &cobra.Command{}
+		err := pluginEnableCmdF(c, cmd, []string{pluginID})
+		s.Require().Nil(err)
+
+		plugins, appErr = s.th.App.GetPlugins()
+		s.Require().Nil(appErr)
+		s.Require().Len(plugins.Active, 1)
+		s.Require().Len(plugins.Inactive, 0)
+
+	})
+
+	s.RunUnauthenticatedClient("error for enable plugin", func(c client.Client) {
+		printer.Clean()
+
+		appErr := s.th.App.DisablePlugin(pluginID)
+		s.Require().Nil(appErr)
+
+		plugins, appErr := s.th.App.GetPlugins()
+		s.Require().Nil(appErr)
+		s.Require().Len(plugins.Active, 0)
+		s.Require().Len(plugins.Inactive, 1)
+
+		cmd := &cobra.Command{}
+		_ = pluginEnableCmdF(c, cmd, []string{pluginID})
+		//err = pluginEnableCmdF(c, cmd, []string{pluginID})
```

Comment:
@mgdelacroix 
pluginEnableCmd prints the error but returns nil
https://github.com/mattermost/mmctl/blob/0476a350cde8e3603f9eef5c53186508a808c655/commands/plugin.go#L140
Whats the expectation here ?

Reply:
It returns `nil` because we don't want to halt the entire operation if a single plugin enable operation fails. You can check the error lines 👍  BTW, please remove commented out lines.